src/main/java/com/hedvig/productPricing/campaign/service/EnableHedvigForeverService.kt

```diff
@@ -1,11 +1,15 @@
 package com.hedvig.productPricing.campaign.service
 
 import com.hedvig.contract.shared.models.objects.Market
+import com.hedvig.productPricing.campaign.domain.model.Campaign
+import com.hedvig.productPricing.campaign.domain.model.Member
 
 interface EnableHedvigForeverService {
   fun enableHedvigForever(
     market: Market,
     members: List<String>
   ): List<String>
+
+  fun getRedeemedHedvigForeverCampaignOrNull(member: Member): Campaign?
```

Comment:
Naming code after marketing is a no no IMO
```suggestion
  fun getRedeemedForeverCampaignOrNull(member: Member): Campaign?
```
```suggestion
  fun getRedeemedCustomReferralCampaignOrNull(member: Member): Campaign?
```
Or something of the like

Reply:
Then shouldn't we rename the service as well? But `CustomReferralCampaign` is not super clear for me what do you think about `EnableReferralsCampaign`